next.clayui.com/content/docs/components/pagination.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: "Pagination"
+description: "Pagination provides horizontal navigation between chunks(pages) of a dataset."
+packageNpm: "@clayui/pagination"
+packageVersion: "3.0.0"
+packageStatus: "Beta"
+---
+
+import {Pagination, PaginationDisabledPages, PaginationEllipsisBuffer, PaginationWithLinks} from '../../../src/components/clay/Pagination';
+
+<div class="nav-toc">
+
+- [Variants and Best Pratices](#variants-and-best-pratices)
+- [API](#api)
+
+</div>
+
+<div class="clay-site-alert alert alert-info">
+Check the <a href="https://liferay.design/lexicon/">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/pagination/">Pagination Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+## Best Pratices
```

Comment:
I'm not sure how useful this section is, just seems like a re-iteration of the API. What do you think @matuzalemsteles?

Reply:
Yes, I think we can limit the information here a bit and we can start doing what we talked about in Slack.